tests/cases/employee_health_portal/case_employee_health_portal_account.yaml

```diff
@@ -316,4 +316,94 @@ TestEmployeeHealthPortalAccountHelp:
           Value: Employee_Health_User_Manual
         - Type: contain
           Var: HELP_PAGE_URL
-          Value: .pdf
\ No newline at end of file
+          Value: .pdf
+#CXTB-109 Employee Health Portal - Request an Appointment for User
+TestEmployeeHealthPortalRequestAnAppointmentForUser:
+  Environment: Settings 
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+  #Calling Login case
+    - Type: case
+      Value: EmployeeHealthPortalLogIn
+  #Going to Request an Appointment
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.tab$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.request_appointment_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.form_modal$
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.create_event_button$
+  # Checking error message
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.employee_health_request_appointment_modal.error_message$
```

Comment:
Should validate the actual error message not just that is displayed

Reply:
I, just removed it, and change the action position according to ticket-[CXTB-109](https://digitalscientists.atlassian.net/browse/CXTB-109)